Core/src/EventData/PrintParameters.cpp

```diff
@@ -10,49 +10,140 @@
 
 #include "Acts/Surfaces/Surface.hpp"
 
+#include <array>
+#include <iomanip>
 #include <ostream>
 
+namespace {
+
+constexpr std::array<std::size_t, 8> kMonotonic = {
+    0, 1, 2, 3, 4, 5, 6, 7,
+};
+
+constexpr std::array<const char*, Acts::eBoundSize> makeBoundNames() {
+  std::array<const char*, Acts::eBoundSize> names = {nullptr};
+  // must be set by index since the order is user-configurable
+  names[Acts::eBoundLoc0] = "l0:";
+  names[Acts::eBoundLoc1] = "l1:";
+  names[Acts::eBoundTime] = "time:";
+  names[Acts::eBoundPhi] = "phi:";
+  names[Acts::eBoundTheta] = "theta:";
+  names[Acts::eBoundQOverP] = "q/p:";
+  return names;
+}
+
+constexpr std::array<const char*, Acts::eFreeSize> makeFreeNames() {
```

Comment:
Could these helper functions also go into an anonymous namespace restricted to this compilation unit? Or do they need to be accessible somewhere else?

Reply:
They already are in an anonymous namespace.